modules/sdk/src/test/scala/org/tessellation/sdk/infrastructure/fork/HandlerSuite.scala

```diff
@@ -0,0 +1,51 @@
+package org.tessellation.sdk.infrastructure.fork
+
+import cats.effect.IO
+
+import org.tessellation.schema.generation.Generation
+import org.tessellation.schema.generators.peerIdGen
+import org.tessellation.schema.gossip._
+import org.tessellation.sdk.config.types.ForkInfoStorageConfig
+import org.tessellation.sdk.domain.fork.{ForkInfo, StoredForkInfo}
+import org.tessellation.sdk.infrastructure.fork.ForkInfoStorageSuite.genStoredForkInfoEntry
+import org.tessellation.sdk.infrastructure.fork.handler.forkInfoHandler
+
+import eu.timepit.refined.auto._
+import io.circe.syntax.EncoderOps
+import weaver.SimpleIOSuite
+import weaver.scalacheck.Checkers
+
+object HandlerSuite extends SimpleIOSuite with Checkers {
+
+  test("recieved gossip gets added to the storage") {
+    val gen = for {
+      forkInfo <- genStoredForkInfoEntry.map(_.forkInfo)
```

Comment:
I don't think you should be accessing a `val` in another suite. Probably better to create a `generators` object and define `genStoredForkInfoEntry` and `genForkInfo` in there.

Reply:
Moved `genStoredForkInfoEntry` to a generators file.